tutorial_sarscov2.md

```diff
@@ -3,17 +3,17 @@ layout: default
 permalink: /tutorial/sars-cov2/
 ---
 
-This tutorial shows you the basic of how to interact with V-pipe.
+This tutorial shows the basics of how to interact with V-pipe.
 
-For the purose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
+For the purpose of this Tutorial, we will work with the `sars-cov2` branch which is [adapted for the SARS-CoV-2 virus]({{ "/sars-cov-2/" | relative_url }}).
```

Comment:
-t-utorial with small t,
we will work -on- the ... branch ?
'SARS-CoV-2 virus'

Sorry, comments are doubling now, was reviewing different commits